specification/communication/data-plane/Identity/stable/2023-08-01/CommunicationIdentity.json

```diff
@@ -0,0 +1,460 @@
+            "type": "string"
+          },
+          {
+            "in": "header",
+            "name": "Repeatability-Request-ID",
+            "description": "Unique identifier of the request.",
+            "type": "string"
+          },
+          {
+            "in": "header",
+            "name": "Repeatability-First-Sent",
+            "description": "Date and time at which the request was first created.",
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Requested scopes for the new token.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CommunicationIdentityAccessTokenRequest"
+            }
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "../../../Common/stable/2021-03-07/common.json#/definitions/CommunicationErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/CommunicationIdentityAccessToken"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Issue an access token with optional idempotent processing using Repeatability-Request-ID and Repeatability-First-Sent headers. If the headers are not provided, the request will be processed in a non-idempotent way.": {
+            "$ref": "./examples/IssueAccessToken.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CommunicationIdentity": {
+      "description": "A communication identity.",
+      "required": [
+        "id"
+      ],
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Identifier of the identity.",
+          "minLength": 1,
+          "type": "string"
+        }
+      }
+    },
+    "CommunicationIdentityAccessToken": {
+      "description": "An access token.",
+      "required": [
+        "expiresOn",
+        "token"
+      ],
+      "type": "object",
+      "properties": {
+        "token": {
+          "description": "The access token issued for the identity.",
+          "minLength": 1,
+          "type": "string"
+        },
+        "expiresOn": {
+          "format": "date-time",
+          "description": "The expiry time of the token.",
+          "type": "string"
+        }
+      }
+    },
+    "CommunicationIdentityAccessTokenRequest": {
+      "required": [
+        "scopes"
+      ],
+      "type": "object",
+      "properties": {
+        "scopes": {
+          "description": "List of scopes attached to the token.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CommunicationIdentityTokenScope"
+          }
+        },
+        "expiresInMinutes": {
+          "format": "int32",
```

Comment:
min, max, default fields are missing, same for line 380

Reply:
Fixed.